templates/default/expat-site.xml.erb

```diff
@@ -147,4 +147,7 @@
         io.hops.hopsworks.expat.migrations.featurestore.storageconnectors.StorageConnectorMigration
         io.hops.hopsworks.expat.migrations.featurestore.trainingdataset.CreateExternalTrainingDatasetInode
     </version-26>
+    <version-30>
```

Comment:
It should be still under <version-26>, we should rename the whole tag once 3.0 is planned for release

Reply:
@maismail do you mean it should be put under `version-26`?

Reply:
yes